test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -472,15 +472,17 @@ def closure():
         else:
             optimizer_state_dict = {}
 
-        optimizer_state_dict = _broadcast_object(
-            optimizer_state_dict,
+        object_list = [optimizer_state_dict]
+        optimizer_state_dict = distributed_c10d.broadcast_object_list(
```

Comment:
curious, can this be called by running `dist.broadcast_object_list`?

Reply:
should be fine, was trying to make it consistent with other file, but makes more sense to use dist.distributed_c10d. directly since it only has one reference :)